packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -57,8 +57,7 @@ export const CreateTopicSection = () => {
     setErrorMsg(null);
   }, [name, featuredInNewPost, editorText]);
 
-  const handleCreateTopic = async (e: React.MouseEvent<HTMLButtonElement>) => {
-    e.preventDefault();
+  const handleCreateTopic = async (_e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
the `handleCreateTopic` will receive form  submitted values (with the same keys we defined in zod validation), i.e 
```
const handleCreateTopic = async (values) => {
// values will have shape = values: { topicName: '', topicDescription: ''}
```

Reply:
Not sure that would work or how, but, regardless, Zod isn't handling all validation.

Reply:
1. if you do not use `e` param anywhere in the function, you can get rid of it
2. the whole point of using CWForm if react-hook-form is that we do not use local state and custom validations. As @mzparacha said, for topic name, we should use state but also `customError` on a field instead of `handleInputValidation`

Reply:
Done!